packages/Interface/src/DTK_CellList.hpp

```diff
@@ -15,13 +15,12 @@
 //---------------------------------------------------------------------------//
 
 #ifndef DTK_CELLLIST_HPP
-#define DTK_CELLLIST_HPP
+#    define DTK_CELLLIST_HPP
```

Comment:
Actually this looks really strange let's not do that

Reply:
OK so no change `IndentPPDirectives`